src/components/DayPickerRangeController.jsx

```diff
@@ -417,6 +411,22 @@ export default class DayPickerRangeController extends React.Component {
       });
     }
 
+    if (minimumNights > 0 && startDate && focusedInput === END_DATE) {
+      modifiers = this.addModifierToRange(
+        modifiers,
+        startDate,
+        startDate.clone().add(minimumNights, 'days'),
+        'blocked-minimum-nights',
+      );
+
+      modifiers = this.addModifierToRange(
```

Comment:
Here is where the blocked modifier will get added back if it has been removed by line 400 unnecessarily. @ljharb